src/query/interpreter.cpp

```diff
@@ -1399,6 +1400,141 @@ PreparedQuery PrepareDumpQuery(ParsedQuery parsed_query, std::map<std::string, T
                        RWType::R};
 }
 
+std::vector<std::vector<TypedValue>> AnalyzeGraphQueryHandler::AnalyzeGraphCreateStatistics(
+    const std::span<std::string> labels, DbAccessor *execution_db_accessor, storage::Storage::Accessor *dba) {
+  auto results = std::vector<std::vector<TypedValue>>();
+  using VertexAccessorIterable = decltype(std::declval<query::DbAccessor>().Vertices(storage::View::OLD));
+  using LPIndex = std::pair<storage::LabelId, storage::PropertyId>;
+  VertexAccessorIterable vertices = execution_db_accessor->Vertices(storage::View::OLD);
+
+  std::map<LPIndex, std::map<storage::PropertyValue, int64_t>> counter;
+
+  // initialize index
+  const auto &indices_info = execution_db_accessor->ListAllIndices();
```

Comment:
If I am not wrong, why do we iterate over labels of indices only, and in case of * analyze only such labels? Because I think there is better option to iterate over indices already created (iterate over such skiplists) and then there is no need to check for label. It could happen that some nodes have multiple labels, but most will probably be in different camps, right?

I am looking for a way to remove completely checks whether label exists.